src/app/pagination/pagination.component.view.spec.ts

```diff
@@ -266,7 +307,26 @@ describe('MzPaginationComponent:view', () => {
             expect(paginationPageButtonArray[i].querySelector('a').innerHTML).toBe((i + 5).toString());
           }
           expect(paginationPageButtonArray.length).toBe(7);
-        })
+        });
+      }));
+
+      it('should go to page 1 when current page is 6 and the total item is change to 50', async(() => {
```

Comment:
If you put a breakpoint in the `firstPage()` method you'll see it is never called ... therefore still not covered